app/controllers/v2/service_bindings_controller.rb

```diff
@@ -38,7 +38,10 @@ def update
         Rails.logger.info("got account: #{account}")
         cmd = "solc --combined-json abi,bin /tmp/simple.sol > /tmp/simple.bin"
         output = `#{cmd}`
-        cmd = "node /var/vcap/packages/cf-containers-broker/pusher.js -p http://#{host}:#{node_port} -a #{account} -x foo /tmp/simple.bin 2>&1"
+        if !parsedParameters.has_key?("contract_args")
+          parsedParameters["contract_args"] = []
+        end
+        cmd = "node /var/vcap/packages/cf-containers-broker/pusher.js -p http://#{host}:#{node_port} -a #{account} -x foo -c #{parsedParameters["contract_args"]} /tmp/simple.bin 2>&1"
```

Comment:
use parsed args in cmd and make sure something exists for you to pass in.

Reply:
agreed. lets validate the passed arguments and error out early. This assumes that users provide an array of args in a JSON format later to be used by `pusher.js` but we need to validate it and possibly reconstruct the array before passing it through.